internal/util/k8sutil/k8sutil.go

```diff
@@ -93,7 +93,11 @@ var dns1123LabelRegexp = regexp.MustCompile("[^a-zA-Z0-9]+")
 // replacing all non-compliant UTF-8 characters with "-".
 func FormatOperatorNameDNS1123(name string) string {
 	if len(validation.IsDNS1123Label(name)) != 0 {
-		return dns1123LabelRegexp.ReplaceAllString(name, "-")
+		// Use - for any of the non-matching characters
+		n := dns1123LabelRegexp.ReplaceAllString(name, "-")
```

Comment:
I know you didn't write the regex for this, but aren't capital letters not allowed in dns names?

Reply:
They are not, but if that regexp didn't contain `A-Z` then all capital letters would be replaced with `-`.